widgetastic_manageiq/__init__.py

```diff
@@ -2083,11 +2277,11 @@ def get_all(self, surf_pages=False):
         if not surf_pages:
             return [self.parent.entity_class(parent=self, name=name) for name in self.entity_names]
         else:
-            items = []
+            entities = []
             for _ in self.paginator.pages():
-                items.extend([self.parent.entity_class(parent=self, name=name)
-                              for name in self.entity_names])
-            return items
+                entities.extend([self.parent.entity_class(parent=self, name=name)
```

Comment:
@mfalesni this would be a case where `parent=self` is no longer needed due to recent WT element fencing updates yes?